node_lambnik/src/tiler/package.json

```diff
@@ -23,12 +23,13 @@
     "bin"
   ],
   "scripts": {
-    "deploy": "yarn transpile && claudia update --set-env POSTGRES_DB=${TILEGARDEN_DB},POSTGRES_HOST=${TILEGARDEN_DB_HOST},POSTGRES_PASSWORD=${TILEGARDEN_DB_PASSWORD},POSTGRES_USER=${TILEGARDEN_DB_USER},DB_TABLE=${DB_TABLE},GEOMETRY_FIELD=${GEOMETRY_FIELD},EPSG=${EPSG} --timeout ${TIMEOUT}",
-    "deploy-new": "yarn transpile && claudia create --region us-east-1 --role lambnik-tiler-dev --api-module bin/api --name tilegarden-dev-${USER} --set-env POSTGRES_DB=${TILEGARDEN_DB},POSTGRES_HOST=${TILEGARDEN_DB_HOST},POSTGRES_PASSWORD=${TILEGARDEN_DB_PASSWORD},POSTGRES_USER=${TILEGARDEN_DB_USER}DB_TABLE=${DB_TABLE},GEOMETRY_FIELD=${GEOMETRY_FIELD},EPSG=${EPSG}",
-    "dev": "nodemon -e js,xml,mss,json --ignore bin/ --exec yarn local",
+    "build-xml": "babel-node src/util/build-xml.js src/config/map-config.mml bin/map-config.xml",
+    "deploy": "yarn transpile && claudia update --timeout ${TIMEOUT}",
+    "deploy-new": "yarn transpile && claudia create --region us-east-1 --role lambnik-tiler-dev --api-module bin/api --name tilegarden-dev-${USER} --timeout ${TIMEOUT}",
+    "dev": "NODE_ENV=development && nodemon -e js,mss,json,mml,mss --ignore bin/ --exec yarn local",
```

Comment:
Ah, that explains some previous behavior I ended up working around. I think I just assumed this was working because the check is for whether or not NODE_ENV==='production', which is set in a different script.